src/lib/util.js

```diff
@@ -197,6 +205,7 @@ function generateRequestHeadersForJWS (
   const ret = Object.fromEntries(
     Object.entries(mappedHeaders).map(([key, value]) => [key.toLowerCase(), value])
   )
+  // clarify, if we need additionalHeaders here (see generateRequestHeaders fn)
```

Comment:
need to remove this comment if no longer needed.

Reply:
done